lib/typeform_data/requestor.rb

```diff
@@ -4,6 +4,12 @@
 module TypeformData
   module Requestor
 
+    RETRY_EXCEPTIONS = [
+      Net::HTTPServiceUnavailable,
+      Errno::ECONNREFUSED,
+      OpenSSL::SSL::SSLError,
```

Comment:
I'd put in a comment here to make it clear that ordinarily one wouldn't retry in this case, but Typeform appears to have transient issues with their SSL that make it necessary. (which is my current understanding)